proto/dwtools/abase/l0.test/Buffer.test.s

```diff
@@ -10532,6 +10532,15 @@ function bufferRetype( test )
   test.shouldThrowErrorSync( () => _.bufferRetype( new BufferRaw( 5 ) ), I8x, 'extra' );
 }
 
+function bufferRetypeWithOffset( test )
+{
+  test.case = 'F64x from BufferRaw with offset to U16x';
```

Comment:
Some cases should have ratio
```js
 src.byteLength / bufferType.BYTES_PER_ELEMENT 
```
equal to an integer. 
For example:
```js
var src = new U8x( [ 1, 2, 3, 4, 5, 6 ] );
var got = _.bufferRetype( src, I16x );
// src.byteLength = 6, bufferTyped.BYTES_PER_ELEMENT = 2 -> 6/2 = 3
```
Some ratios should be float numbers
```js
var src = new U8x( [ 1, 2, 3, 4, 5, 6 ] );
var got = _.bufferRetype( src, F32x );
// src.byteLength = 6, bufferTyped.BYTES_PER_ELEMENT = 4 -> 6/4 = 1.5 => 1;
```

Implement a few test cases like 
```js
var src = new U8x( [ 1, 2, 3, 4, 5, 6 ] );
var got = _.bufferRetype( src, F64x );
// src.byteLength = 6, bufferTyped.BYTES_PER_ELEMENT = 8 -> 6/8 = 0.75 => 0
```